rancher-common/provider.tf

```diff
@@ -54,7 +54,7 @@ provider "rancher2" {
 
 # Rancher2 administration provider
 provider "rancher2" {
-  version = "~> 1.7"
+  version = "1.7.3"
```

Comment:
Could you please updated to use v1.8.3 instead??

Reply:
I'll make another PR to lock to 1.8.3 since I've tested thoroughly with 1.7.3 and want to make sure this is dangling for as little time as possible.